frontend/src/components/organisms/TableFilters/index.jsx

```diff
@@ -41,10 +67,16 @@ const TableFilter = (props) => {
     updateSelectedFilter,
   } = props;
 
-  const category =
-    filterData?.length > 0 &&
-    categoryFilterValue &&
-    filterData.filter((item) => item.name === categoryFilterValue)?.[0]?.key;
+  const getFilterID = (name) => {
+    if (!(tableFilters?.length > 0)) {
+      return "";
+    }
+    const filterResults = tableFilters.filter((item) => item.name === name);
+    if (filterResults.length == 0) {
```

Comment:
Instead of checking the length of the `filterResults` and returning early, it can be handled in the `filter` function with optional chaining.   
`const filterResults = tableFilters.filter((item) => item.name === name)?.[0].value`

Reply:
I switched it to the simplified line that you proposed.